File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-05/non-plagiarized/09/t05.kt
package IR

import java.util.*

/*
* To change this license header, choose License Headers in Project Properties.
* To change this template file, choose Tools | Templates
* and open the template in the editor.
*/
/**
 *
 * @author 51BE04B11C5F2F5AD96A396632DC3F76
 */

object t05 {
    @JvmStatic
    fun main(args: Array<String>) {
        val sc = Scanner(System.`in`)
        print("Enter an integer: ")
        val num = sc.nextInt()
        reverse(num)
    }

    fun reverse(number: Int) {
        val tmpNumber = number.toString()
        for (i in tmpNumber.length - 1 downTo -1 + 1) {
            print(tmpNumber[i])
        }
        println("")
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-05/non-plagiarized/09/t05.py
import sys

def reverse(number):
    tmp_number = str(number)
    for i in range(len(tmp_number) - 1, -1, -1):
        print(tmp_number[i], end='')
    print()

if __name__ == '__main__':
    num = int(input("Enter an integer: "))
    reverse(num)

